config/vufind/config.ini

```diff
@@ -535,10 +535,10 @@ ils_encryption_key = false
 ; This is the algorithm used to encrypt and decrypt catalog passwords.
 ; A symmetrical encryption algorithm must be used.
 ; You can use openssl_get_cipher_methods() to see available options on your system.
-; Common choices: blowfish (default), aes, cast, sm4
+; Common choices: blowfish (deprecated -- do not use), aes (recommended), cast, sm4
 ; If you want to convert from one algorithm to another, run this from $VUFIND_HOME:
 ;   php public/index.php util switch_db_hash oldhash:oldkey (or none) newhash:newkey
-;ils_encryption_algo = "blowfish"
+ils_encryption_algo = "aes"
```

Comment:
If we're turning this on be default, will the system prompt for or generate a random key for the cipher?

Reply:
We're not turning encryption on by default, we're just setting a default encryption algorithm to use if the setting is enabled.

Reply:
(The encrypt_ils_password setting above still defaults to false, and that's the key thing).